tests/e2e/Services/Account/AccountCustomClientTest.php

```diff
@@ -1500,6 +1500,15 @@ public function testCreateJWT(): array
 
         $this->assertEquals(200, $response['headers']['status-code']);
 
+        $response = $this->client->call(Client::METHOD_GET, '/account/sessions/current', array_merge([
+            'origin' => 'http://localhost',
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+            'x-appwrite-jwt' => $jwt,
+        ]));
+
+        $this->assertEquals(200, $response['headers']['status-code']);
+
```

Comment:
@stnguyen90 I was thinking to add a test for checking the expiry time condition for JWT to validate the changes of #8000 I did above.

Should we add that? If yes, is there any recommended way here to change the default expiry time, or use some other session creation endpoint which allows setting the expiry time?

Reply:
Can you make an API call to update the project auth settings and change the session expiry before creating the session?

Otherwise, you can try doing a manual test and add screenshot(s) as evidence.

Reply:
@stnguyen90 Thanks for the idea of calling project auth API to change the expiry time!

Did that, and added the test to test if JWT properly respects session expiry time!